tests/Selenium2Config.php

```diff
@@ -62,6 +62,18 @@ public function skipMessage($testCase, $test): ?string
             return 'Checking status code is not supported.';
         }
 
+        if (
+            'Behat\Mink\Tests\Driver\Js\JavascriptTest' === $testCase
+            && 'testDragDropOntoHiddenItself' === $test
+        ) {
+            $seleniumVersion = $_SERVER['SELENIUM_VERSION'] ?? null;
+            $browser = $_SERVER['WEB_FIXTURES_BROWSER'] ?? null;
```

Comment:
Shouldn't we use `getenv()` in those two cases?

Reply:
Please see https://github.com/minkphp/webdriver-classic-driver/issues/44 . 

To summarize:

1. when we use `<env...` in `phpunit.xml` then:
   * using `getenv` in PHP works
   * overriding from GitHub Actions works
   * using `$_SERVER` doesn't work
2. when we use `<server...` in `phpunit.xml` then:
   * using `getenv` in PHP doesn't work
   * overriding from GitHub Actions doesn't work
   * using `$_SERVER` works
3. when we don't use `<env...` or `<server...` in `phpunit.xml` then:
   * using `getenv` in PHP - ??? (haven't tested)
   * overriding from GitHub Actions works
   * using `$_SERVER` works

Ideally, we should switch to `<env` + `getenv` in the whole test suite, but that is BC break for drivers. Still waiting for @stof response in the above-mentioned issue.

Reply:
Understood, too bad.

Reply:
I'll wait for a resolution on the above-mentioned issue. Maybe this will lead to something good.